### PR TITLE
NVLI-216 created the detail APi

### PR DESCRIPTION
**Pull request by dishabhadra**
@piyuesh23 : Kindly do the code review.

### CODE REVIEW

File: docroot/modules/custom/nvli_custom_search_api/src/Plugin/rest/resource/NvliNodeDetail.php [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+   *
+   * @return \Drupal\rest\ResourceResponse
+   *   The HTTP response object.
+   *
+   * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+   *   Throws exception expected.
+   */
+  public function get($nid) {
+    // Load the node.
+    $node = node_load($nid);
```
Comment (piyuesh23) [inline docroot/modules/custom/nvli_custom_search_api/src/Plugin/rest/resource/NvliNodeDetail.php]: Can we rather use Entitymanager to load nodes. This is deprecated.

**Comment by dishabhadra** [comment]
@piyuesh23  : Changes are done.

**Comment by piyuesh23** [comment]
@dishabhadra Merge conflicts here.

**Comment by dishabhadra** [comment]
@piyuesh23 : conflicts are resolved.

**Pull request merged**